eng/Testing.props

```diff
@@ -1,17 +1,12 @@
 <Project>
-  <PropertyGroup Condition="'$(UseVSTestRunner)' != 'true'">
-    <_QuarantinedTestRunAdditionalArgs>-trait "quarantined=true"</_QuarantinedTestRunAdditionalArgs>
-    <_NonQuarantinedTestRunAdditionalArgs>-notrait "quarantined=true"</_NonQuarantinedTestRunAdditionalArgs>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(UseVSTestRunner)' == 'true'">
-    <_QuarantinedTestRunAdditionalArgs>--filter "quarantined=true"</_QuarantinedTestRunAdditionalArgs>
-    <_NonQuarantinedTestRunAdditionalArgs>--filter "quarantined!=true"</_NonQuarantinedTestRunAdditionalArgs>
+  <PropertyGroup>
+    <_QuarantinedTestRunAdditionalArgs>--filter-trait "quarantined=true"</_QuarantinedTestRunAdditionalArgs>
+    <_NonQuarantinedTestRunAdditionalArgs>--filter-not-trait "quarantined=true"</_NonQuarantinedTestRunAdditionalArgs>
   </PropertyGroup>
 
   <PropertyGroup>
     <BlameHangTimeout Condition="'$(BlameHangTimeout)' == ''">10m</BlameHangTimeout>
```

Comment:
seems this is 12min elsewhere. maybe we should pick one.

Reply:
There is a bit of inconsistency in how tests are setup currently, with too many runsettings, and `dotnet test` calls that specify things that are already in runsettings (and the command-line options then take precedence over runsettings).